lib/metrics/prometheus.js

```diff
@@ -47,49 +50,56 @@ class PrometheusMetric {
     }
 
     increment(amount, labels) {
-        this._handleStaticLabels(labels);
-        labels = labels.map(this._normalize);
-        this.metric.labels.apply(this.metric, labels).inc(amount);
+        let updatedLabels = [...labels];
+        this._handleStaticLabels(updatedLabels);
```

Comment:
Would actually be nicer if you clone them inside handleStaticLabels and return new labels.

then here you would be like `const updatedLabels = this._handleStaticLabels( labels );`. More readable.